### ISSUE TITLE
Crash when trying to upgrade libc++ from AUR

### ISSUE THREAD

**Issue by untitaker** [Kwpolska/pkgbuilder#54]
Thanks for pkgbuilder. I found the following bug:

```
$ pb -S --asdeps libc++
==> Building libc++...
  -> aur/libc++ 5.0.1-1 (123 votes) [installed: 5.0.0-1]
  ->     A new implementation of the C++ standard library, targeting C++11.
==> Cloning the git repository...
Already up to date.
==> Checking dependencies...
Traceback (most recent call last):
  File "/usr/bin/pb", line 11, in <module>
    load_entry_point('pkgbuilder==4.2.14', 'console_scripts', 'pb')()
  File "/usr/lib/python3.6/site-packages/pkgbuilder/wrapper.py", line 34, in main
    pkgbuilder.wrapper.wrapper()
  File "/usr/lib/python3.6/site-packages/pkgbuilder/wrapper.py", line 311, in wrapper
    pbmain(pbargs + pbpkgnames)
  File "/usr/lib/python3.6/site-packages/pkgbuilder/__main__.py", line 349, in main
    DS.pkginst, pkgnames)
  File "/usr/lib/python3.6/site-packages/pkgbuilder/build.py", line 52, in auto_build
    build_result = build_runner(pkgname, performdepcheck, pkginstall)
  File "/usr/lib/python3.6/site-packages/pkgbuilder/build.py", line 493, in build_runner
    deps = depcheck(depends, pkg)
  File "/usr/lib/python3.6/site-packages/pkgbuilder/build.py", line 272, in depcheck
    fdep).groups()[0]
  File "/usr/lib/python3.6/re.py", line 172, in match
    return _compile(pattern, flags).match(string)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 619, in _parse
    source.tell() - here + len(this))
sre_constants.error: multiple repeat at position 5

$ pb --version
PBWrapper   v0.5.4               
PKGBUILDer  v4.2.14
pacman      v5.0.2
pyalpm      v0.8
```

**Comment by Kwpolska** [comment]
Since when can AUR characters contain `+` in their names? I’ll release v4.2.15 with a fix soon.

**Comment by Kwpolska** [comment]
Fixed in commit 00005cf.
